src/Homework5/IMDB.py

```diff
@@ -0,0 +1,45 @@
+'''В файле хранятся данные с сайта IMDB.
+Скопированные данные хранятся в файле ./data5/ ratings.list.
+Откройте и прочитайте файл(если его нет необходимо вывести ошибку).
+Найдите ТОП250 фильмов и извлеките заголовки.
+Программа создает 3 файла  top250_movies.txt –
+названия файлов, ratings.txt – гистограмма рейтингов,
+years.txt – гистограмма годов.
+'''
+line_starts_with_new = False
+count_movies = 0
+counter = {}
+y_counter = {}
+try:
+    with open('ratings.list', 'r', encoding="ISO-8859-1") as f:
+        with open('top250_movies.txt', 'w') as top250:
+            with open('ratings.txt', 'w') as movie_ratings:
+                with open('years.txt', 'w') as years:
+                    for line in f:
```

Comment:
overall looks too complex

Reply:
Rewritten a little, but to me looks ok, because by opening all the files at once it simplifies the code and we don't have to repeat some parts of the code, we just write what's needed on the go in corresponding files.

Reply:
we can move repeatable parts to functions